assets/javascripts/discourse/components/chat-live-pane.js

```diff
@@ -189,9 +189,13 @@ export default Component.extend({
   },
 
   @bind
-  onScrollHandler() {
+  onScrollHandler(e) {
     cancel(this.stickyScrollTimer);
-    this.stickyScrollTimer = discourseDebounce(this, this.onScroll, 100);
+    this.stickyScrollTimer = discourseDebounce(
+      this,
+      () => this.onScroll(e),
```

Comment:
Does it work? I don't think using an anonymous function and debounce works as it always create a new context. Try using a high value instead of 100 and see if it's debounced.

Reply:
Yes, it works! Unless I missed something, the fat arrow should bind `this` to the context given to debounce.

Reply:
Tried with a high value, and `discourseDebounce` works as expected this way 👍

Reply:
OK I see good :+1: thanks